README.md

```diff
@@ -3,127 +3,139 @@
 
-- [Random X](https://github.com/tevador/RandomX)
+_Algorithms to determine how a block is hashed._
+
+-   [Random X](https://github.com/tevador/RandomX)
 
-***
+* * *
 
 ### Mining Software
-*3rd party software to assist mining assets.*
 
-- [BLOC MultiCurrency GUI Miner](https://github.com/furiousteam/BLOC-GUI-Miner)
-- [Scala GUI Miner](https://github.com/scala-network/GUI-miner)
-- [Vulkan XMR Miner](https://github.com/enerc/VulkanXMRMiner)
-- [XMR-Stak](https://github.com/fireice-uk/xmr-stak)
-- [XMRig](https://github.com/xmrig/xmrig)
-  - [XMRig - Nvidia](https://github.com/xmrig/xmrig-nvidia)
-  - [XMRig - AMD](https://github.com/xmrig/xmrig-amd)
+_3rd party software to assist mining assets._
 
-***
+-   [BLOC MultiCurrency GUI Miner](https://github.com/furiousteam/BLOC-GUI-Miner)
+-   [Scala GUI Miner](https://github.com/scala-network/GUI-miner)
+-   [Vulkan XMR Miner](https://github.com/enerc/VulkanXMRMiner)
+-   [XMR-Stak](https://github.com/fireice-uk/xmr-stak)
+-   [XMRig](https://github.com/xmrig/xmrig)
+-   [XMRig - Nvidia](https://github.com/xmrig/xmrig-nvidia)
+-   [XMRig - AMD](https://github.com/xmrig/xmrig-amd)
+
+* * *
 
 ### Pool Software
-*Base templates to guide you in creating a pool.*
 
-- [Dvandal - cryptonote-nodejs-pool](https://github.com/dvandal/cryptonote-nodejs-pool)
-- [FunkyPenguin - cryptonote-nodejs-pool](https://github.com/funkypenguin/cryptonote-nodejs-pool)
-- [MuscleMan - cryptonote-nodejs-pool](https://github.com/muscleman/cryptonote-nodejs-pool)
+_Base templates to guide you in creating a pool._
+
+-   [Dvandal - cryptonote-nodejs-pool](https://github.com/dvandal/cryptonote-nodejs-pool)
+-   [FunkyPenguin - cryptonote-nodejs-pool](https://github.com/funkypenguin/cryptonote-nodejs-pool)
+-   [MuscleMan - cryptonote-nodejs-pool](https://github.com/muscleman/cryptonote-nodejs-pool)
 
-***
+* * *
 
 ### Web Wallet
-*Access your assets anywhere.*
 
-- [Monero: MyMonero](https://github.com/mymonero) - [Link](https://wallet.mymonero.com/)
-- [Masari: Web Wallet](https://github.com/masari-project/masari-webwallet) - [Link](https://www.masariwallet.com/)
+_Access your assets anywhere._
+
+-   [Monero: MyMonero](https://github.com/mymonero) - [Link](https://wallet.mymonero.com/)
+-   [Masari: Web Wallet](https://github.com/masari-project/masari-webwallet) - [Link](https://www.masariwallet.com/)
 
-***
+* * *
 
 ### Misc.
-*Applications, games, resources and other things built and/or maintained by CryptoNote developers.*
 
-- [DecentraChat](https://github.com/lithyriolu/decentrachat) - [LithyRiolu](https://github.com/lithyriolu) - [The Lithe Project&trade;](https://github.com/lithe-project)
+_Applications, games, resources and other things built and/or maintained by CryptoNote developers._
 
-***
+-   [DecentraChat](https://github.com/lithyriolu/decentrachat)
+-   [LithyRiolu](https://github.com/lithyriolu)
+-   [The Lithe Project™](https://github.com/lithe-project)
```

Comment:
And these 3 new lines too.

The layout will look like this in the future hopefully
- Appname - Author - Project
- Appname - Author - Project